src/package_details.html

```diff
@@ -0,0 +1,39 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Package Details</title>
+    <link rel="stylesheet" href="../assets/css/main.css">
+    <script src="chef_workstation.js"></script>
```

Comment:
I cannot find in the electron documentation now but when [upgrading Electron to the latest version](https://github.com/chef/chef-workstation-app/pull/74) it recommended switching to `require` vs `script src=`. The closest thing I can find now [this StackOverflow](https://stackoverflow.com/a/54330076) explanation.

It should look like:
```
<script>
  const chef_workstation = require('./chef_workstation.js')
<script>
...
<script>document.write(chef_workstation.getInstalledGemVersion('chef-cli'))</script>
```

Reply:
Cool, I'll change it! 💯